emails/models.py

```diff
@@ -262,7 +267,7 @@ def make_domain_address(user_profile, address=None, made_via_email=False):
             )
 
         address_contains_badword = False
-        if address is None:
+        if not address:
```

Comment:
Doing `not address` because address value can be an empty string which would fail this check.